queue_services/common/src/entity_queue_common/service_utils/__init__.py

```diff
@@ -14,19 +14,20 @@
 """Callbacks and utility functions used to support the service loop."""
 from typing import Callable
 
-import stan
+from stan.aio.client import Client as StanClient
+from stan.aio.client import Msg as StanMessage
```

Comment:
Fixed this issue when importing as "stan". 
https://github.com/nats-io/stan.py/issues/48#issuecomment-764486455